Neos.Flow/Classes/Aop/Pointcut/PointcutMethodNameFilter.php

```diff
@@ -126,6 +126,10 @@ public function matches($className, $methodName, $methodDeclaringClassName, $poi
                     return false;
                 }
                 break;
+            default:
+                if ($methodDeclaringClassName !== null && $this->reflectionService->isMethodPrivate($methodDeclaringClassName, $methodName)) {
+                    throw new Aop\Exception('The method "' + $methodDeclaringClassName + ':' + $methodName + '" adviced by "' + $this->aspectClassName + ':' + $this->pointcutMethodName + '" is private. Private methods can not be adviced. Please change the scope to at least protected or explicitly specify the visibility you want to match.', 1593013922);
```

Comment:
`Aop\Exception` does not resolve to a FQCN, shouldn't this just be `Exception`?

Also `$this-> aspectClassName` and `$this-> pointcutMethodName` are not defined in this class..!?

Reply:
D'oh, yeah - I had this inside the `Pointcut` first, then thought that to be the wrong place and just moved it into the `PointcutMethodNameFilter`. Looking at it again, I'm not sure anymore about this. Exception should probably be thrown in the constructor instead of on match, then rethrown in the `PointcutExpressionParser` with the `sourceHint`?

Reply:
Mh, not sure, that whole proxy building is still Greek to me..
I was about to ditch my version for yours but #2131 seems to do the trick just fine – shall we just go for that?

Reply:
Looks a bit cleaner than the throw+catch/rethrow that would be necessary here (to get the source hints), so sure. Let's ditch this aproach